build.gradle

```diff
@@ -173,6 +173,20 @@ configurations {
     }
 }
 
+def findJarByName(Configuration config, String name) {
+    config.find { File file -> file.name.startsWith(name)}
+}
+
+task setupGatkLaunch(type: Copy) {
+    def gatkJar = { findJarByName(configurations.compile, 'gatk') }
```

Comment:
I'm not sure how to check that actually...  I think it won't be a problem if it does because it won't have a gatk-launch.